src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java

```diff
@@ -119,11 +120,11 @@ public MaskPasswordsOutputStream(OutputStream logger, @CheckForNull Collection<S
 
     @Override
     protected void eol(byte[] bytes, int len) throws IOException {
-        String line = new String(bytes, 0, len);
+        String line = new String(bytes, 0, len, "UTF-8");
```

Comment:
I suppose the more conservative change would be to restore the original code, and add

```java
@SuppressFBWarning(value="whatever", justification="TODO should probably be using Run.charset")
```